pypit/arcoadd.py

```diff
@@ -304,7 +305,7 @@ def median_ratio_flux(spec, smask, ispec, iref, nsig=3., niter=5, **kwargs):
     # Ratio
     med_flux = fluxes[iref,allok] / fluxes[ispec,allok]
     # Clip
-    mn_scale, med_scale, std_scale = sigma_clipped_stats(med_flux, sigma=nsig, iters=niter, **kwargs)
+    mn_scale, med_scale, std_scale = astropy.stats.sigma_clipped_stats(med_flux, sigma=nsig, iters=niter, **kwargs)
```

Comment:
Was this another circular import?
Just checking..

Reply:
No, not a circular import.  Just a syntax preference.  The module loads both scipy.stats and astropy.stats, so I just wanted to be clear where this comes from.